### ISSUE TITLE
SSH wrapper doesn't seem to do anything useful

### ISSUE THREAD

**Issue by tomasruud** [warpdotdev/Warp#1926]
### Local Shell and Version

zsh 5.8.1

### Remote Shell and Version

bash 4.4.20(1)-release

### Operating System and Version

MacOS Monterey 12.6

### Warp Version

v0.2022.09.27.08.11.stable_00

### I have tried commenting out my system & user rcfiles and warp still will not bootstrap

- [X] Yes

### Screenshots

How Warp looks before running SSH command:
<img width="594" alt="image" src="https://user-images.githubusercontent.com/5978545/194280165-b789edce-2fd4-41fe-84c2-b490962cac3c.png">

How it looks once logged in:
<img width="484" alt="image" src="https://user-images.githubusercontent.com/5978545/194280302-0eacd6ad-1134-4e23-a069-16e72afc644f.png">

### Include shell xtrace output

```
WARP_DEBUG_MODE=1 ssh example.com
bash-4.4$
```

**Comment by dannyneira** [comment]
Hi @tomasruud , So sorry to hear about this. So the SSH wrapper adds the blocks and other functionality to your SSH sessions but can produce issues if you have some custom prompt in the remote server ~/.bashrc | ~/.zshrc files.

My guess is that you normally have a [custom prompt and that Warp is unable to parse it. ](https://docs.warp.dev/features/prompt#ps1-compatibility-table)

You can check whether it’s something in your dotfiles by [setting up clean configs:](https://docs.warp.dev/help/known-issues#configuring-and-debugging-your-rc-files)
Set
`ZDOTDIR=/`
in your
`~/.zshenv`
file. This forces Zsh to run with zero configs.

You can disable parts of your dotfiles just for Warp by using this conditional statement:

```sh

if [[ $TERM_PROGRAM != "WarpTerminal" ]]; then
##### WHAT YOU WANT TO DISABLE FOR WARP - BELOW

# Whatever you want to disable

##### WHAT YOU WANT TO DISABLE FOR WARP - ABOVE
fi

```
We have a list of incompatible tools here: <https://docs.warp.dev/help/known-issues#list-of-incompatible-tools>
And more info on custom prompts here: <https://docs.warp.dev/features/prompt>

Let me know if these help in any way to resolve your issue.

**Comment by tomasruud** [comment]
I don't have a custom prompt, and it seems like the issue still persists when I run Zsh with zero config. However, I tested with a different server, and it seems like the issue might be related to the server I'm trying to connect to. I've removed bashrc files etc from that server, but I guess it might be something related to the version the server is running?

**Comment by dannyneira** [comment]
Yeah, it sounds like could be the case just on that server, might have some plugin or prompt that Warp can't parse correctly. 

Check our [SSH GitHub milestone ](https://github.com/warpdotdev/Warp/issues/578)to see if your issue has already been reported. 

Meanwhile, you can disable the SSH wrapper by navigating to `Settings > Features`. Blocks will stop working but with it disabled, SSH should work as normal.

**Comment by tomasruud** [comment]
@dannyneira is there any debug info from the server or my local machine that I can provide you with to see if this is related to one of the other issues? I can't really say I see my issue listed there, but it might be I'm not understanding the problem properly.

**Comment by dannyneira** [comment]
Hi @tomasruud, My mistake, just noticed you're using bash so the clean config won't work, try moving the .bashrc file on your remote machine temporarily and let us know if that helps.

* Move/apply changes with `mv ~/.bashrc /tmp/.bashrc`
* Open new ssh session, if it fixes the issues, if it does then you need to put something in that file inside the conditional statement.

```sh
if [[ $TERM_PROGRAM != "WarpTerminal" ]]; then
 ##### WHAT YOU WANT TO DISABLE FOR WARP - BELOW

 # Whatever you want to disable

 ##### WHAT YOU WANT TO DISABLE FOR WARP - ABOVE
 fi
 ```
* Then move back with `mv /tmp/.bashrc ~/.bashrc`

**Comment by tomasruud** [comment]
I tried doing this, but the result is still the same. I'm unsure what else could be the issue.
